docs/src/main/asciidoc/_building.adoc

```diff
@@ -1,5 +1,26 @@
 == How to Build Spring Cloud Contract
 
+=== Cloning the repository on Windows
+
+While cloning this project on *windows*, some files in the git repository may exceed the Windows maximum file path limit of 255 characters, which may
+result in improper/half checked out repository.
+
+* To resolve this the `*core.longPaths=true*` git option should be set:
```

Comment:
Change to

To resolve this issue, you can set the `core.longPaths` attribute to `true` or clone the Spring Cloud Contract repository.
The following example shows how to set the `core.longPaths` attribute to `true`: